guacamole-docker/bin/start.sh

```diff
@@ -659,9 +659,15 @@ associate_header() {
 ##
 start_guacamole() {
 
+    # User-only writable CATALINA_BASE
+    export CATALINA_BASE=$GUACAMOLE_HOME/tomcat
```

Comment:
I don't think `CATALINA_BASE` should be a subdirectory of `GUACAMOLE_HOME`. The contents and purpose of `GUACAMOLE_HOME` are dictated by the webapp. It contains specific things in a specific structure and shouldn't be used as a general-purpose storage location.

Reply:
Thank you very much for your feedback @mike-jumper 👍
You're right, I modified accordingly.